src/applications/disability-benefits/686c-674/config/chapters/household-income/householdIncome.js

```diff
@@ -0,0 +1,18 @@
+import { netWorthCalculation, netWorthTitle } from './helpers';
+
+export const schema = {
+  type: 'object',
+  properties: {
+    householdIncome: {
```

Comment:
Should this schema use the property in vets-json-schema? - https://github.com/department-of-veterans-affairs/vets-json-schema/pull/534/files#diff-4379fe63910601ab909e1565040fe385c2be632aff74669202761e88f58149feR900